src/components/Dropdown.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {
+  useCallback,
+  useMemo,
+  useState,
+  useRef,
+  useEffect,
+} from 'react';
+import debounce from 'lodash.debounce';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+interface Props {
+  people: Person[];
+  delay: number;
+  onSelected?: (user: Person) => void,
+}
+
+export const Dropdown: React.FC<Props> = React.memo(({
```

Comment:
I would say `Autocomplete` or `AutocompletePeople` would be a better name

Reply:
since we return a <div className='dropdown'>, I named the component this way + its easy to understand what kind of stuff we get on our page

For me, it would be hard to understand what kind of component "insides" we are getting, with a name "Autocomplete"

So I would prefer to leave the name as it is